pkg/lib/config/identity.go

```diff
@@ -302,7 +302,7 @@ var _ = Schema.Add("OAuthSSOProviderConfig", `
 		"key_id": { "type": "string" },
 		"team_id": { "type": "string" },
 		"app_type": { "$ref": "#/$defs/OAuthSSOWeChatAppType" },
-		"account_id": { "type": "string" },
+		"account_id": { "type": "string", "format": "wechat_account_id"},
```

Comment:
The error message displayed in portal is not very informative. Can we localize the this error?

Reply:
Opened issue https://github.com/authgear/authgear-server/issues/902